### PR TITLE
My tests

### PR DESCRIPTION
**Pull request by aakiselev94**

### CODE REVIEW

File: src/test/java/com/epam/javacourses2016/Task17/Task17Test.java [hunk 1]
```diff
@@ -1,112 +0,0 @@
...
-import org.testng.annotations.Test;
-import java.math.RoundingMode;
-import java.text.DecimalFormat;
-import java.util.*;
-
-/**
- * Created by kodoo on 13.11.16.
- */
-
-public class Task17Test {
```
Comment (nkarulin) [inline src/test/java/com/epam/javacourses2016/Task17/Task17Test.java:17]: You should not remove others developers work resolving conflicts)
Keep the test, extend it

**Comment by aakiselev94** [comment]
Ready!

**Comment by nkarulin** [comment]
You're still deleting Task17 test due to github diffs. Package renaming won't help

**Comment by nkarulin** [comment]
Ok, master won't be updated, no need to sync with it. Thanks!

**Pull request closed without merging**